src/doc-editors/ServerGroupsEditor.vue

```diff
@@ -454,13 +492,13 @@ export default defineComponent({
       }
     },
 
-    async deleteDoc() {
+    async deleteServerGroupDocByName() {
       this.setLoadingDocStatus(true)
       this.isDeleteLoading = true
-      const serverGroupText = this.titles['sites-singular']
+      const ServerGroupText = this.titles['sites-singular']
```

Comment:
Javascript variables should be camelCase (Need to fix uses as well)

```suggestion
      const serverGroupText = this.titles['sites-singular']
```